### DISCUSSION TITLE
How to monitor business data in SQL Server use Hertzbeat ?

### DISCUSSION THREAD

**Discussion by mralanchan** [apache/hertzbeat#2766] [General]
We want monitor the business data, e.g. Sales Order Progress (Created, WIP, Completed), is there any config sample to reference?

**Comment by zuobiao-zhou** [comment]
Hi, you can reference [this link](https://hertzbeat.apache.org/zh-cn/docs/advanced/extend-jdbc) to customize [sql server monitor template](https://github.com/apache/hertzbeat/blob/master/manager/src/main/resources/define/app-sqlserver.yml)
